packages/hs-react-ui/package.json

```diff
@@ -123,7 +124,9 @@
     "webpack": "webpack --config webpack.config.js",
     "dts": "tsc --declaration --outDir dist/types/ --emitDeclarationOnly --declarationMap --allowJs false --checkJs false",
     "pre-push": "yarn run lint-ci && yarn run test && yarn run clean && yarn run webpack && yarn run dts",
-    "test": "rm -rf ./coverage/* && jest --collectCoverage=true",
+    "test": "rm -rf ./coverage/* && jest",
```

Comment:
`Test` should run both `test:coverage` and `test:badges` for our build process. The `test` script gets called during our CICD pipeline

Reply:
I will append `&& jest-coverage-badges` to the **_test_** script so the badges are updated in the CI pipeline.